src/Mews.Fiscalizations.All/Mews.Fiscalizations.All.csproj

```diff
@@ -10,7 +10,7 @@
     <RepositoryUrl>https://github.com/MewsSystems/fiscalizations</RepositoryUrl>
     <Icon>https://raw.githubusercontent.com/msigut/eet/master/receipt.png</Icon>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <PackageVersion>1.0.4</PackageVersion>
+    <PackageVersion>1.0.5</PackageVersion>
```

Comment:
Shouldn't we also bump Czechia?

Reply:
@marektresnak already did in the previous PR, but i forgot about All :D